nativeApp/src/screens/admin_views/Volunteers/VolunteersPage/index.tsx

```diff
@@ -19,27 +27,73 @@ type Props = {
  */
 
 /*
-* Component
-*/
+ * Component
+ */
+const Volunteers: FC<NavigationFocusInjectedProps & Props> = ({ navigation }) => {
+  // Redux
+  const dispatch = useDispatch();
 
+  useEffect(() => {
+    dispatch(loadVolunteers());
+  }, []);
 
-const Volunteers: FC<NavigationFocusInjectedProps & Props> = ({ navigation }) => {
+  const volunteers = useSelector(selectOrderedVolunteers, shallowEqual);
+  const volunteersRequestStatus = useSelector(selectVolunteersStatus, shallowEqual);
+
+  // React State
   const [deleteModalVisibility, toggleDeleteModalVisibility] = useToggle(false);
+  const [activeCard, setActiveCard] = useState(null);
+
+  // Handlers
+  const onDelete = (id: number) => {
+    toggleDeleteModalVisibility();
+    setActiveCard(id);
+  };
+
+  const onConfirm = () => {
+    /*
+    * TODO:
+    * - dispatch redux action deleteVolunteerAndUpdate(activeCard.id)
```

Comment:
This action would be an example of an action that can change the state of the volunteers on the backend, and should therefore mean the list is marked as `stale`.

(Although, alternatively, we know what effect this action has, so we can just remove that user from the store, but I was just highlighting an example of my previous comment).